build_npm.ts

```diff
@@ -0,0 +1,31 @@
+// ex. scripts/build_npm.ts
+import { build, emptyDir } from "https://deno.land/x/dnt/mod.ts";
+
+await emptyDir("./npm");
+
+await build({
+  entryPoints: ["./src/mod.ts"],
+  outDir: "./npm",
+  shims: {
+    // see JS docs for overview and more options
+    deno: true,
+  },
+  package: {
+    // package.json properties
+    name: "@slack/deno-slack-sdk",
+    version: Deno.args[0],
+    description: "Test deno sdk dpendencies",
```

Comment:
Let's modify this general package description to be more descriptive for external users. Once the package is published to npm, developers will see as the description for this package,.